library/account/src/main/kotlin/org/cru/godtools/account/GodToolsAccountManager.kt

```diff
@@ -73,15 +75,21 @@ class GodToolsAccountManager @VisibleForTesting internal constructor(
         createAccount: Boolean,
         onResponse: (LoginResponse) -> Unit,
     ): ActivityResultLauncher<AccountType> {
+        val context = LocalContext.current
```

Comment:
Instead of using Local context, inject the application context into the account manager Singleton.